bookwyrm/activitypub/book.py

```diff
@@ -37,6 +37,7 @@ class Book(BookData):
     publishedDate: str = ""
 
     cover: Document = None
+    preview_image: Document = None
```

Comment:
When the database field is converted into an activitypub field, it is automatically switched from snake_case to camelCase, so this should be previewImage here.

That said, does it make the most sense for these to federate along with the book, or for them to be generated dynamically when the book is created locally? If instances have custom themes, you will end up with the theme of the instance that created the book when it someone links to the book on your instance

Reply:
I see, I was using the wrong model type! Thank you